spec.html

```diff
@@ -22945,17 +22978,21 @@ <h1>Static Semantics: ExportedBindings</h1>
       <emu-clause id="sec-exports-static-semantics-exportednames">
         <h1>Static Semantics: ExportedNames</h1>
         <emu-see-also-para op="ExportedNames"></emu-see-also-para>
-        <emu-grammar>ExportDeclaration : `export` `*` FromClause `;`</emu-grammar>
+        <emu-grammar>ExportDeclaration : `export` ExportFromClause FromClause `;`</emu-grammar>
+        <emu-alg>
+          1. Return the ExportedNames of ExportFromClause.
+        </emu-alg>
+        <emu-grammar>ExportFromClause : `*`</emu-grammar>
         <emu-alg>
           1. Return a new empty List.
         </emu-alg>
-        <emu-grammar>
-          ExportDeclaration : `export` ExportClause FromClause `;`
-
-          ExportDeclaration : `export` ExportClause `;`
-        </emu-grammar>
+        <emu-grammar>ExportFromClause : NamedExports</emu-grammar>
         <emu-alg>
-          1. Return the ExportedNames of |ExportClause|.
+          1. Return the ExportedNames of NamedExports.
```

Comment:
Same here: production chaining should take care of this.

Reply:
Same question. Is it reasonable to leave it in for clarity? Otherwise I agree it's unnecessary.